cmd/skipper/main.go

```diff
@@ -271,7 +274,7 @@ func init() {
 	flag.BoolVar(&enableRatelimiters, "enable-ratelimits", false, enableRatelimitUsage)
 	flag.Var(&ratelimits, "ratelimits", ratelimitUsage)
 	flag.StringVar(&openTracing, "opentracing", "noop", opentracingUsage)
-	flag.StringVar(&pluginDir, "plugindir", ".", pluginDirUsage)
+	flag.StringVar(&pluginDir, "plugindir", "", pluginDirUsage)
```

Comment:
pluginDirUsage should have a deprecation mark, can you check if this is in the usage string?